_dashboards/management/connect-prometheus.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: Connecting Prometheus to OpenSearch
+parent: Data sources
+nav_order: 20
+---
+
+# Connecting Prometheus to OpenSearch
+
+This documentation focuses on using the OpenSeach Dashboards interface to connect to Prometheus. 
+
+## Prerequisites and permissions
+
+Make sure you followed and configured [Prerequisites]({{site.url}}{{site.baseurl}}/dashboards/management/data-sources/#prerequisites) and [Permissions]({{site.url}}{{site.baseurl}}/dashboards/management/data-sources/#permissions).
```

Comment:
I believe there may be boilerplate for this elsewhere that we should be consistent with. It should read something like "Make sure that you have configured all necessary prerequisites and permissions for/in order to..."